### PR TITLE
Move container output to the rich print system.

### PR DESCRIPTION
**Pull request by fnando**
### What

```console
$ stellar network container start testnet
ℹ️ Starting testnet network
✅ Started container
🔎 You can see logs by running `stellar network container logs testnet`
ℹ️ To stop this container, use `stellar network container stop testnet`

$ stellar network container stop testnet
ℹ️ Stopping testnet container
✅ Container has been stopped
```

### Why

So we only have one way of printing rich output everywhere.

### Known limitations

N/A

### CODE REVIEW

File: cmd/soroban-cli/src/commands/network/container/start.rs [hunk 1]
```diff
@@ -164,40 +180,42 @@ impl Cmd {
...
-        );
-        println!("{log_message}");
-    }
+    fn print_instructions(&self) {
+        let container_name = self.container_name().get_external_container_name().clone();
+        let additional_flags = self.args.container_args.get_additional_flags().clone();
+        let tail = format!("{container_name} {additional_flags}");
+
+        self.print.searchln(format!(
+            "You can see logs by running `stellar network container logs {}`",
```
Comment (leighmcculloch) [inline cmd/soroban-cli/src/commands/network/container/start.rs:197]: The 'you' is unnecessary:
```suggestion
            "Watch logs with `stellar network container logs {}`",
```

File: cmd/soroban-cli/src/commands/network/container/start.rs [hunk 2]
```diff
@@ -164,40 +180,42 @@ impl Cmd {
...
-    fn print_stop_message(&self) {
-        let stop_message =
-            format!(
-            "ℹ️ To stop this container run: stellar network container stop {container_name} {additional_flags}",
-            container_name = self.container_name().get_external_container_name(),
-            additional_flags = self.container_args.get_additional_flags(),
-        );
-        println!("{stop_message}");
+        self.print.infoln(format!(
+            "To stop this container, use `stellar network container stop {}`",
```
Comment (leighmcculloch) [inline cmd/soroban-cli/src/commands/network/container/start.rs:202]:
```suggestion
            "Stop the container with `stellar network container stop {}`",
```

File: cmd/soroban-cli/src/commands/network/container/stop.rs [hunk 3]
```diff
@@ -49,10 +56,9 @@ impl Cmd {
                     Error::ContainerStopFailed(e)
                 }
             })?;
-        println!(
-            "✅ Container stopped: {}",
-            container_name.get_external_container_name()
-        );
+
+        print.checkln("Container has been stopped");
```
Comment (leighmcculloch) [inline cmd/soroban-cli/src/commands/network/container/stop.rs:60]:
```suggestion
        print.checkln("Container stopped");
```

**Review by leighmcculloch — APPROVED** [review]
Couple text suggestions otherwise lgtm

**Review by elizabethengelman — APPROVED** [review]
Lgtm! 👏

**Pull request merged**